PoSSapi/src/Domain/Entities/User.cs

```diff
@@ -5,12 +5,17 @@ namespace PoSSapi.Domain.Entities;
 public class User
 {
     public Guid Id { get; set; }
-    public int BusinessId { get; set; }
+    public Business Business { get; set; }
     public string Name { get; set; }
     public string Surname { get; set; }
     public int Age { get; set; }
     public string Username { get; set; }
     public string Email { get; set; }
     public string Password { get; set; }
     public UserType UserType { get; set; }
+    public string Position { get; set; }
+    public decimal Salary { get; set; }
+    public ICollection<Order> Orders { get; set; }
+    public ICollection<Payment> Payments { get; set; }
+    public ICollection<Reservation> Reservations { get; set; }
```

Comment:
Why is this needed here, doesn't it provide duplicate information that could be gotten in another way?